curriculum/challenges/english/01-responsive-web-design/applied-visual-design/decrease-the-opacity-of-an-element.english.md

```diff
@@ -24,7 +24,7 @@ Set the <code>opacity</code> of the anchor tags to 0.7 using <code>links</code>
 ```yml
 tests:
   - text: Your code should set the <code>opacity</code> property to 0.7 on the anchor tags by selecting the class of <code>links</code>.
-    testString: assert(getComputedStyle($('.links')[0]).opacity == '0.7');
+    testString: assert(/\.links{\S*opacity:0?\.70*(;\S*}|})/.test($('style').text().replace(/\s/g ,'')));
```

Comment:
```suggestion
    testString: assert(/\.links{\S*opacity:\s?0?\.70*(;\S*}|})/.test($('style').text().replace(/\s/g ,'')));
```

Reply:
How does this look?

Reply:
@thecodingaviator No, if anything it would need to account for something like:
```
.links {
  opacity: .7
}
```

The assert below would be more flexible:
```
assert(/\.links{\s*opacity\s*:\s*0?\.70*\s*(;[\s\S]*}|\s*})/.test($('style').text()));
```

Reply:
We need to account for things like this:

![image](https://user-images.githubusercontent.com/34807532/69912298-5045a400-144d-11ea-9be0-b1cd7debfd70.png)

![image](https://user-images.githubusercontent.com/34807532/69912300-5d629300-144d-11ea-9a87-ae0eb842d5d4.png)